File: c294.c
#include <stdio.h>

int main(void) {
    int num[3];
    int tmp;
    while (scanf("%d %d %d", &num[0], &num[1], &num[2])) {
        for (int i = 0; i < 3; i++) {
            for (int j = i + 1; j < 3; j++) {
                if (num[i] > num[j]){
                    tmp = num[i];
                    num[i] = num[j];
                    num[j] = tmp;
                }
            }
        }
        
        printf("%d %d %d\n", num[0], num[1], num[2]);
        if (num[0] + num[1] <= num[2])
            printf("No\n");
        else if (num[0]*num[0] + num[1]*num[1] < num[2]*num[2])
            printf("Obtuse\n");
        else if (num[0]*num[0] + num[1]*num[1] == num[2]*num[2])
            printf("Right\n");
        else
            printf("Acute\n");
    }
    return 0;
}

File: c294.py
# -*- coding: utf-8 -*-
num = []
num = list(map(int, input().split()))
num.sort()
print(" ".join([str(i) for i in num]))
if (num[0] + num[1] <= num[2]):
    print("No")
elif (num[0]*num[0] + num[1]*num[1] < num[2]*num[2]):
    print("Obtuse")
elif (num[0]*num[0] + num[1]*num[1] == num[2]*num[2]):
    print("Right")
else:
    print("Acute")